package.json

```diff
@@ -38,80 +38,80 @@
   "devDependencies": {
     "@faker-js/faker": "^9.3.0",
     "@shopify/eslint-plugin": "^46.0.0",
-    "@types/better-sqlite3": "^7.6.9",
-    "@types/cookie-parser": "^1.4.7",
+    "@types/better-sqlite3": "^7.6.12",
+    "@types/cookie-parser": "^1.4.8",
     "@types/cors": "^2.8.17",
     "@types/detect-character-encoding": "^0.7.2",
-    "@types/express": ">=4.17.21",
-    "@types/express-session": "^1.18.0",
+    "@types/express": "^5.0.0",
```

Comment:
does this need to stay in sync with the express major version? (looks like we're still on `"express": "^4.21.2",` below)

Reply:
Changed back to Express 4 on the types... Will raise a new PR for Express 5.0 shortly.